spec/rspec/mocks/diffing_spec.rb

```diff
@@ -83,6 +83,21 @@
       end
     end
 
+    if RUBY_VERSION >= "3" && RSpec::Support::RubyFeatures.kw_args_supported?
+      it "display keyword hashes" do
+        with_unfulfilled_double do |d|
+          expect(d).to receive(:foo).with(**expected_hash)
```

Comment:
```
spec/rspec/mocks/diffing_spec.rb:89: syntax error, unexpected tPOW, expecting ')'
            expect(d).to receive(:foo).with(**expected_hash)
```
is not something that is hard to work around, see an example https://github.com/rspec/rspec-support/blob/d04c8aef72d030fc354902f1e1b486b5c21e0732/spec/rspec/support/method_signature_verifier_spec.rb#L276

Reply:
Oh wow, I didn't realize rspec was still compatible with 1.8. 

I'll fix this.

Reply:
Done.